### DISCUSSION TITLE
Hi! I wonder if it's possible to enable this also for the featured image of a post?

### DISCUSSION THREAD

**Discussion by Jaethon** [archetyped/simple-lightbox#1027] [Q&A]
Currently, images inside the post content work as expected and pop up in the lightbox.  But the featured image at the top of the page doesn't have a link and isn't available to SLB.

Is there a way that I can add it?  Even by adding some code in a snippet?

Thanks for any feedback!

**Comment by Jaethon** [comment]
Hey guys... I figured it out... thanks!  I changed single.php in my child theme...

**Comment by Jaethon** [comment]
Oh.. I used this code : 

			$full_image_url = wp_get_attachment_image_src( get_post_thumbnail_id(), 'full' );
			$newcontent = '<a name="'. $p->id . '" class="post-id" data-id="'. $p->id . '" href="' . $full_image_url[0] . '" title="' . the_title_attribute( 'echo=0' ) . '">' . get_the_post_thumbnail( $post_id, 'full' ) . '</a>';
			if ( function_exists('slb_activate') ) {
				$newcontent = slb_activate($newcontent);
				}
			echo $newcontent;

**Comment by archetyped** [comment]
Glad you found a solution.  For future reference, [SLB's Activation documentation](https://github.com/archetyped/simple-lightbox/wiki/Activation) contains instructions on manually activating content, including a streamlined code snippet for integrating the a post's featured image with SLB.
